src/storage/src/generated/protos/storage/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Code generated by sidekick. DO NOT EDIT.
+
+[package]
+name        = "google-cloud-storage-v2-code-generation"
+version     = "0.0.0"
+description = "Prost and Tonic generator for google-cloud-storage-v2"
+publish     = false
+build       = "build.rs"
+edition     = "2021"
```

Comment:
non-blocking Q: do we have to use 2021? can it be 2024?

Reply:
Yes, we could change that.